carbon-design-system/carbon-for-ibm-dotcom
# feat(tab-extended-vertical): add "Vertical" Orientation

### Related Ticket(s)

DDS Consulting Issue: https://github.com/carbon-design-system/carbon-for-ibm-dotcom/issues/6614
HC JIRA Ticket: https://jsw.ibm.com/browse/HC-2066

### Description
- This PR adds in the "Orientation" prop that allows adopters to set the orientation of the Extended Tabs layout component to display as `horizontal (default)` or `vertical`.

### Design & Functional Specs
- https://github.com/carbon-design-system/carbon-for-ibm-dotcom-website/wiki/Tabs-extended#22-orientation 

![Screen Shot 2021-08-05 at 3 45 33 PM](https://user-images.githubusercontent.com/1815714/128411793-98546629-d0ec-4027-a09d-f181d4d01b70.png)



### Changelog

**New Components/Props**

- `Orientation` - this prop/attribute allows adopters to set the orientation of the Extended Tabs component to horizontal (default) or vertical.

**Removed**

- N/A

### Note

- The layout component/band that's designed for the GEO page in AEM requires `card-link` component, however, this particular card component doesn't have the same functional specs as the `<dds-card-group-item>` component, which can make it difficult to assemble within AEM.
- We might want to possible extend the `<dds-card-link>` component to the following:
```
<dds-card-group>
   <dds-card-group-link-item href="https://example.com">
   </dds-card-group-link-item>
</dds-card-group>
```

### Testing Instructions
- Visit the `Tab Extended` layout component - `/?path=/story/components-tabs-extended--default`
- Navigate to the "Knobs" tab and set `Orientation` to `vertical`
- Ensure that the "Vertical" tabs component matches with the design specs for all breakpoints.
